### DISCUSSION TITLE
When will a new resilience4j-micronaut version with support for Micronaut 4 be released?

### DISCUSSION THREAD

**Discussion by abarth911** [resilience4j/resilience4j#2061] [Q&A] [answered]
Hello together,

I've saw that the support for Micronaut 4 was already merged into master and I've wanted to ask when there'll be a new version for resilience4j-micronaut.

I would really appreciate if there'll be a new release with Micronaut 4 support soon.

Thank you!

**Comment by RobWin** [comment, marked as answer]
Hi, yes sry.
Quite busy working on AI projects.
I hope I find some time soon to release a new version.

**Reply by jmera** [reply]
@RobWin is there anything we can do to help with this release?

**Reply by RobWin** [reply]
I need an AI assistant who is writing releases notes for me, checking open issues and releasing Resilience4j for me :)
I áctually need to find some time to write the RELEASE notes and check if there is something important missing for a next release.

**Comment by tmalinakis-vng** [comment]
Hi, we are also awaiting for this. I noticed there was an attempt to release version 2.2.0 but it failed due to an issue while publishing?

**Comment by RobWin** [comment]
2.2.0 has been released

**Reply by abarth911** [reply]
Thank you
